### PR TITLE
A few minor changes to iterate on the amiable e-mail

### PR DESCRIPTION
**Pull request by sihil**
A few changes:
 - make gmail / inbox treat the e-mails more sensibly rather than bundling them together (by putting the date and owner in subject)
 - change the columns of information in the e-mail to make it more relevant / useful
 - add a note on how the owners are derived
 - add links to the SSA and AMI columns

screen shot 2017-06-20 at 18 54 32

### CODE REVIEW

File: app/views/email.scala.html [hunk 1]
```diff
@@ -16,34 +17,49 @@
...
+                            <td style="padding: 5px 5px;">@instance.instanceName</td>
+                            <td style="padding: 5px 5px;">
+                                @maybeAmi.map { ami =>
+                                    <a href="@{amiableUrl + routes.AMIable.ami(ami.imageId).path}">@ami.imageId</a>
+                                }
+                            </td>
+                            <td style="padding: 5px 5px;">@maybeAmi.flatMap(_.creationDate).map { i => @DateUtils.daysAgo(i)
+                                days
+                            }</td>
+                        </tr>
```
Comment (TBonnin) [inline app/views/email.scala.html]: ❤️

File: app/views/email.scala.html [hunk 2]
```diff
@@ -16,34 +17,49 @@
...
                 }
                 <tr bgcolor="#212121" style="color: #FFFFFF !important; padding: 30px 30px 30px 30px;">
                     <td colspan="7" style="padding: 15px 5px;">
-                        You are receiving this email because you are listed as the Owner of the following stacks:
+                        You are receiving this email because you are listed as the Owner of the stacks listed below.
+                        This is determined by the Owners in prism (see
+                            <a href="https://github.com/guardian/prism/blob/master/app/data/Owners.scala">GitHub</a>).
```
Comment (TBonnin) [inline app/views/email.scala.html]: 👍

File: nginx/amiable.sh [hunk 3]
```diff
@@ -5,13 +5,7 @@
...
 nginxHome=`nginx -V 2>&1 | grep "configure arguments:" | sed 's/[^*]*conf-path=\([^ ]*\)\/nginx\.conf.*/\1/g'`
 
-SYSTEM=$(uname -s)
-if [ $SYSTEM == "Darwin" ]; then
-    # Mac OS X platform
-    confDir=servers
-elif [ $SYSTEM == "Linux" ]; then
-    confDir=sites-enabled
-fi
+confDir=sites-enabled
```
Comment (TBonnin) [inline nginx/amiable.sh]: I think nginx installed on OSX with `brew` stores conf files in a `servers` dir. At least that how it is on my laptop

Comment (sihil) [inline nginx/amiable.sh]: Hmmm, not on mine - but I'll revert this part as I didn't mean it to be in the PR.

**Pull request merged**